rust/types/src/api_types.rs

```diff
@@ -113,21 +114,41 @@ impl ChromaError for ResetError {
     }
 }
 
-#[derive(Serialize)]
+#[derive(Serialize, ToSchema)]
```

Comment:
nit: it might make sense to gate these derivations behind a feature flag

Reply:
Talked to @codetheweb and we decided it was ok to do this as a follow-up task

Reply:
Please cut ticket